forced_photometry/multiband_photometry.md

```diff
@@ -210,29 +210,96 @@ spitzer['cloud_access'] = [(f'{{"aws": {{ "bucket_name": "irsa-mast-tike-spitzer
+    verbose : bool, optional
+        If True, print status messages. Default is False.
+
+    Raises
+    ------
+    ValueError
+        If neither 'fname' nor 'name' columns are found in `data_table`.
+
+    Notes
+    -----
+    - The function checks if a file already exists before downloading it.
+    - It creates the target directory if it does not exist.
+    - Uses `fornax.get_data_product()` to retrieve the data handler.
+
+    Examples
+    --------
+    >>> data_table = [{'fname': 'file1.fits', 'access_url': 'https://example.com/file1.fits'},
+    ...               {'fname': 'file2.fits', 'access_url': 'https://example.com/file2.fits'}]
+    >>> fornax_download(data_table, 'my_data', verbose=True)
+    Skipping file1.fits: already exists.
+    Downloaded and saved file2.fits to data/my_data
+    Download process complete.
+    """
+    # Define the absolute path of the target directory
+    data_directory = os.path.join("data", data_subdirectory)
+    os.makedirs(data_directory, exist_ok=True)  # Ensure the directory exists
+
+    # Check which filename column exists
+    filename_column = None
+    for col in ['fname', 'name']:
+        if col in data_table.colnames:
+            filename_column = col
+            break  # Use the first found column
+
+    if not filename_column:
+        raise ValueError("Error: Neither 'fname' nor 'name' columns found in the data table.")
 
-    data_directory = f"data/{data_subdirectory}"
-    os.chdir(data_directory)
     for row in data_table:
-        if fname_filter is not None and fname_filter not in row['fname']:
+        filename = os.path.basename(row[filename_column])  # Extract filename
+        file_path = os.path.join(data_directory, filename)  # Full file path
+
+        # Skip download if file already exists
+        if os.path.exists(file_path):
+            if verbose:
+                print(f"Skipping {filename}: already exists.")
             continue
+
+        # Apply filename filter, if provided
+        if fname_filter is not None and fname_filter not in filename:
+            continue
+
+        # Download the file
         handler = fornax.get_data_product(row, 'aws', access_url_column=access_url_column, verbose=verbose)
         temp_file = handler.download()
-        # on-prem download returns temp file path, s3 download downloads file
+
+        # Move the downloaded file if a local file path is returned
         if temp_file:
-            shutil.move(temp_file, os.path.basename(row['fname']))
+            shutil.move(temp_file, file_path)
+            if verbose:
+                print(f"Downloaded and saved {filename} to {data_directory}")
 
-    os.chdir(working_dir)
+    if verbose:
+        print("Download process complete.")
 ```
 
 ```{code-cell} ipython3
 fornax_download(spitzer, access_url_column='sia_url', fname_filter='go2_sci',
-                data_subdirectory='IRAC', verbose=False)
+                data_subdirectory='IRAC', verbose=True)
```

Comment:
Do we really need to have these verbose by default? If it's only a few files then fine, but I would avoid filling up the notebook/rendered HTMLs with these verbose printouts here and below

Reply:
no, that was me working on the function and forgetting to switch them back to False. Thanks for noticing.